packages/ketcher-polymer-editor-react/src/Editor.tsx

```diff
@@ -208,7 +210,9 @@ function Editor({ theme, togglerComponent }: EditorProps) {
     });
     editor?.events.mouseOnMoveMonomer.add((e) => {
       handleClosePreview();
-      handleOpenPreview(e);
+      if (e.buttons !== 1 || !noPreviewTools.includes(activeTool)) {
```

Comment:
Can we move `e.buttons !== 1` to constant like `isLeftClick` (am I right about this number?). Everything else looks good time from code perspective